### PR TITLE
NIFI-11910: Clarified some points in docs

### PR DESCRIPTION
**Pull request by markap14**
# Summary

[NIFI-00000](https://issues.apache.org/jira/browse/NIFI-00000)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [ ] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [ ] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [ ] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [ ] Pull Request based on current revision of the `main` branch
- [ ] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java [hunk 1]
```diff
@@ -78,12 +58,32 @@
...
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.regex.Pattern;
+
 @SideEffectFree
 @SupportsBatching
 @InputRequirement(Requirement.INPUT_REQUIRED)
 @Tags({"Unpack", "un-merge", "tar", "zip", "archive", "flowfile-stream", "flowfile-stream-v3"})
 @CapabilityDescription("Unpacks the content of FlowFiles that have been packaged with one of several different Packaging Formats, emitting one to many "
-        + "FlowFiles for each input FlowFile")
+        + "FlowFiles for each input FlowFile. Support formats are TAR, ZIP, and FlowFile Stream packages.")
```
Comment (ChrisSamo632) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java:86]:
```suggestion
        + "FlowFiles for each input FlowFile. Supported formats are TAR, ZIP, and FlowFile Stream packages.")
```

**Review by ChrisSamo632 — COMMENTED** [review]
Looks good @markap14, just one very minor thing I'd suggest changing, although it's probably not a blocker

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java (hunk 1 above)
Comment (markap14) [reply nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java:86]: Thanks. Nothing like submitting a PR to fix docs, and then introducing new typos 😂 This is why we have reviewers :) Will push a fix.

**Review by ChrisSamo632 — CHANGES_REQUESTED** [review]
Looks good now @markap14, except for merge conflicts

**Review by ChrisSamo632 — APPROVED** [review]

**Comment by ChrisSamo632** [comment]
LGTM +1, merging

**Pull request closed without merging**